spec/charts/bar_spec.rb

```diff
@@ -29,5 +29,21 @@
       ]
     )
   end
+
+  it 'merged options and more_options correctly' do
+    options = { plotOptions: { bar: { dataLabels: { position: "top" } } } }
+    chart = described_class.new(bindings, data, options)
+
+    expect(chart.instance_variable_get(:@options)[:plotOptions]).to eq(
```

Comment:
I'm not excited about digging into the @options instance variable here, but not sure if you want to expose that options var or not.

Reply:
Well, I never thought I would need to expose that. But for the sake of the test, I guess it should be exposed. Let me merge this first.